src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -164,18 +169,20 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
 
     public async showPipelineFiles(): Promise<void> {
         this.filesToCommit.forEach(async (file) => {
-            await this.localGitRepoHelper.addContentToFile(file.content, file.absPath);
+            await this.localGitRepoHelper.addContentToFile2(file.content, file.absPath);
             await vscode.window.showTextDocument(vscode.Uri.file(file.absPath), { preview: false });
         });
     }
 
     public setQueuedPipelineUrl(provisioningConfiguration: ProvisioningConfiguration, inputs: WizardInputs) {
         const commitId = (provisioningConfiguration.result.pipelineConfiguration as CompletePipelineConfiguration).commitId;
         this.queuedPipelineUrl = `https://github.com/${inputs.sourceRepository.repositoryId}/commit/${commitId}/checks`;
+        this.committedWorkflow = `https://github.com/${inputs.sourceRepository.repositoryId}/commit/${commitId}`;
     }
 
     public async getFileToCommit(draftPipelineConfiguration: DraftPipelineConfiguration): Promise<void> {
         let destination: string;
+        this.tmpDirectoryPath = fse.mkdtempSync(os.tmpdir());
```

Comment:
This function seems to be also available in `fs` package. Any advantage of using `fse`?

Reply:
`fs-extra` basically extends the functionally ``fs`` provides. It contains all ``fs`` module function plus some additional function